force-app/main/service/DonationHistoryService.cls

```diff
@@ -35,16 +35,12 @@
 public with sharing class DonationHistoryService {
     
     static final String NPSP_CREDIT_ROLES = 'NPSP_OppsSoftCreditRoles';
-    private List<OpportunityContactRole> hardCredits;
-    private List<OpportunityContactRole> softCredits;
-    private List<OpportunityContactRole> partialSoftCredits;
+    private List<OpportunityContactRole> credits;
```

Comment:
This variable is always an empty list and can be removed.

Reply:
variable removed!